### PR TITLE
IBX-1648: Added redirect when the trash form has any error

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-1648
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by micszo** [comment]
@mateuszdebinski does the second commit intentionally contain a different JIRA id?

**Comment by mateuszdebinski** [comment]
> @mateuszdebinski does the second commit intentionally contain a different JIRA id?

Thanks, corrected, it's wasn't intentional

**Review by mikadamczyk — APPROVED** [review]

File: src/bundle/Controller/LocationController.php [hunk 1]
```diff
@@ -354,6 +354,11 @@ public function trashAction(Request $request): Response
             if ($result instanceof Response) {
                 return $result;
             }
+
+            $data = $form->getData();
+            $location = $data->getLocation();
+
+            return $this->redirectToLocation($location);
```
Comment (ViniTou) [inline src/bundle/Controller/LocationController.php]: What if `$location` is null?

Comment (mateuszdebinski) [reply src/bundle/Controller/LocationController.php]: Is it at all possible for `$location` to be empty when operating on location?

but of course, I can add 'if' if it doesn't find a location then move it to the trash as it is now

Comment (mateuszdebinski) [reply src/bundle/Controller/LocationController.php]: Ok never mind I checked and it's possible. Added condition to check if the location is not null

**Review by ViniTou — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

**Pull request merged**